docs/manual-EN/manual-introduction.md

```diff
@@ -0,0 +1,75 @@
+# Nebula Graph R201910_beta User Manual
+
+### About this manual
+
+This is the Nebula Graph User Manual. It documents Nebula Graph R201910_beta. It may include documentation of features of Nebula Graph versions that have not yet been released. For information about which versions have been released, see  Nebula Graph Release Notes.
+<!-- Release Notes not added yet. -->
+
+Because this manual serves as a reference, it does not provide general instruction on  graph database concepts. It also does not teach you how to use your operating system or command-line interpreter.
+
+Nebula Graph is under constant development, and the User Manual is updated frequently as well.
+
+The User Manual source files are written in  Markdown format. The HTML version is produced automatically, primarily using mkdocs. For information about mkdocs, see https://www.mkdocs.org/
+
+If you have questions about using Nebula Graph, join the [Nebula Graph Community Slack](https://join.slack.com/t/nebulagraph/shared_invite/enQtNjIzMjQ5MzE2OTQ2LTM0MjY0MWFlODg3ZTNjMjg3YWU5ZGY2NDM5MDhmOGU2OWI5ZWZjZDUwNTExMGIxZTk2ZmQxY2Q2MzM1OWJhMmY#"). If you have suggestions concerning additions or corrections to the manual itself, please do not hesitate to open an issue on [GitHub](https://github.com/vesoft-inc/nebula/issues).
+
+This manual was originally written and maintained by the Nebula Graph Documentation Team.
+
+### Typographical and syntax conventions
+This manual uses certain typographical conventions:
+- `Fixed width`
+
+    A fixed-width font is used in syntax, code examples, system output, and file names.
+- **Bold**
+
+    Bold typeface indicates commands or characters the user types, provides emphasis, or the names of user interface elements.
+- *Italic*
+
+    Italic typeface indicates the title of a document, or signifies new terms.
+- `UPPERCASE fixed width`
+
+     Keywords in syntax and code examples are almost always shown in upper case. Although shown in uppercase, you can type keywords in either uppercase or lowercase.
+
+### Overview of Nebula Graph
```

Comment:
I think we can put this section in a new file.
`About this manual`.md
`Overview of Nebula Graph`.md